dbms/src/Storages/KVStore/CMakeLists.txt

```diff
@@ -24,6 +24,7 @@ add_headers_and_sources(kvstore ./Read)
 add_subdirectory (./MultiRaft/Disagg)
 
 add_library(kvstore ${kvstore_headers} ${kvstore_sources})
+target_include_directories(kvstore PRIVATE ${TiFlash_SOURCE_DIR}/contrib/GmSSL/include)
```

Comment:
Do we use `USE_GM_SSL` here?

Reply:
work like this: FileEncryption.h -> AESCTRCipherStream.h -> AESCTRCipher.h -> gmssl/sm4.h